mysql-test/suite/rocksdb.rpl/t/multiclient_2pc.test

```diff
@@ -23,7 +19,7 @@ connect (con2, localhost, root,,);
 # has been recovered and commited
 connection con1;
 --echo 'con1'
---exec echo "restart" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--exec echo "$restart_parameters" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
 SET SESSION debug="d,crash_commit_after_log";
 SET DEBUG_SYNC='rocksdb.prepared SIGNAL parked WAIT_FOR go';
 --error 0,2013
```

Comment:
No idea, original Facebook code, easy enough to change to modern semantics and drop the 0.